NeoMathEngine/src/MathEngineDnnDropout.h

```diff
@@ -35,4 +35,19 @@ struct CMathEngineDropoutDesc : public CDropoutDesc {
 	CFloatHandleVar Mask;
 };
 
+// DropoutDesc without mask for usage on CUDA
+struct CCudaMathEngineDropoutDesc : public CDropoutDesc {
```

Comment:
This class is for CUDA only. It's better to move somewhere to `NeoMathEngine/src/GPU/CUDA/*`. 
E.g. in front of its' ctor definition in CudaMathEngineDnnDropout.cu